src/DssSpell.sol

```diff
@@ -73,25 +82,89 @@ contract DssSpellAction is DssAction {
 
-        // Huntingdon Valley (HVBank) Vault Drawdown
+        // ----------------------------- RWA Draws -----------------------------
+        // https://vote.makerdao.com/polling/QmQMDasC#poll-detail
+        // Weekly Draw for HVB
+        address RWA009_A_URN = DssExecLib.getChangelogAddress("RWA009_A_URN");
         RwaUrnLike(RWA009_A_URN).draw(RWA009_DRAW_AMOUNT);
 
+        // --------------------------- Rates updates ---------------------------
+        // https://vote.makerdao.com/polling/QmfMRfE4#poll-detail
+
+        // Reduce Stability Fee for    ETH-B   from 4% to 3.75%
+        DssExecLib.setIlkStabilityFee("ETH-B", THREE_SEVEN_FIVE_PCT_RATE, true);
+
+        // Reduce Stability Fee for    WSTETH-A   from 2.50% to 2.25%
+        DssExecLib.setIlkStabilityFee("WSTETH-A", TWO_TWO_FIVE_PCT_RATE, true);
+
+        // Reduce Stability Fee for    WSTETH-B   from 0.75% to 0%
+        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_PCT_RATE, true);
+
+        // Reduce Stability Fee for    WBTC-B   from 4.00% to 3.75%
+        DssExecLib.setIlkStabilityFee("WBTC-B", THREE_SEVEN_FIVE_PCT_RATE, true);
+
+        // Increase Stability Fee for  GUNIV3DAIUSDC1-A   from 0.01% to 0.02%
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC1-A", ZERO_ZERO_TWO_PCT_RATE, true);
+
+        // Increase Stability Fee for  GUNIV3DAIUSDC2-A   from 0.05% to 0.06%
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC2-A", ZERO_ZERO_SIX_PCT_RATE, true);
+
+        // Increase Stability Fee for  UNIV2DAIUSDC-A   from 0.01% to 0.02%
+        DssExecLib.setIlkStabilityFee("UNIV2DAIUSDC-A", ZERO_ZERO_TWO_PCT_RATE, true);
+
+        // ------------------------ Debt Ceiling updates -----------------------
+        // https://vote.makerdao.com/polling/QmfMRfE4#poll-detail
+
+        // Increase the line for              GUNIV3DAIUSDC2-A from   1 billion to 1.25 billion DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("GUNIV3DAIUSDC2-A",                   1250 * MILLION);
+        // Increase the line for              GUINV3DAIUSDC1-A from 750 million to 1 billion DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("GUNIV3DAIUSDC1-A",                   1 * BILLION);
+        // Increase the line for              MANA-A           from  15 million to 17 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("MANA-A",                             17 * MILLION);
+
+        // ----------------------- Activate SocGen Vault -----------------------
+        // NOTE: ignore in goerli
+        // call hope() on RWA008_A_OUTPUT_CONDUIT
+        // https://forum.makerdao.com/t/socgen-forge-ofh-granting-final-permission-after-the-onboarding/17033
+        address RWA008_A_OPERATOR       = 0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439;
+        address RWA008_A_OUTPUT_CONDUIT = DssExecLib.getChangelogAddress("RWA008_A_OUTPUT_CONDUIT");
+        HopeLike(RWA008_A_OUTPUT_CONDUIT).hope(RWA008_A_OPERATOR);
```

Comment:
Probably should move this closer to the HVB draw-down.

Reply:
I was following the spreadsheet layout/order from governance. I think that makes it easier to review rather than having to jump around to compare things